server/routes/genesetEnrichment.ts

```diff
@@ -79,7 +79,7 @@ async function run_gsea(path, data) {
 		const _stderr: any[] = []
 		// spawn python process
 		let python_path = 'python3'
-		if (serverconfig.py_path) python_path = serverconfig.py_path
+		if (serverconfig.python) python_path = serverconfig.python
```

Comment:
@robinpaul85 What I meant in my previous comment is this:
- in `server/src/serverconfig.js`, add the last line
```js
if (!serverconfig.Rscript) serverconfig.Rscript = 'Rscript'
if (!serverconfig.gfServer) serverconfig.gfServer = 'gfServer'
if (!serverconfig.gfClient) serverconfig.gfClient = 'gfClient'
if (!serverconfig.python) serverconfig.python = 'python3'
```

- in `genesetEnrichment.ts`, do not use `python_path`, use `serverconfig.python` directly. This will guarantee that all nodejs script use the same tool binary, instead of each code file setting its own default binary path, we're trying to avoid that
```ts
const sp = spawn(serverconfig.python, [path])
```